Update ru_ru.lang

Bytegm opened the pull request:
minor bugs fixed

KnightMiner commented:
Since I don't really understand Russian, mind explaining what you changed in a comment for future reference?

I see you are targeting 0.3.2, though my next release will probably be from 0.4.x code. I changed a couple lang keys which you changed back, specifically `tcomplement.config.general.oreToIngotRatio` is the correct key now. On a related note, are you planning on adding the rest of the 0.4 features, or should I pull it as is? Mostly, 0.4.x added the high oven blocks and added some book pages which can be translated.

Bytegm commented:
> Since I don't really understand Russian, mind explaining what you changed in a comment for future reference?
> 
> I see you are targeting 0.3.2, though my next release will probably be from 0.4.x code. I changed a couple lang keys which you changed back, specifically `tcomplement.config.general.oreToIngotRatio` is the correct key now. On a related note, are you planning on adding the rest of the 0.4 features, or should I pull it as is? Mostly, 0.4.x added the high oven blocks and added some book pages which can be translated.

Bytegm commented:
> Since I don't really understand Russian, mind explaining what you changed in a comment for future reference?
> 
> I see you are targeting 0.3.2, though my next release will probably be from 0.4.x code. I changed a couple lang keys which you changed back, specifically `tcomplement.config.general.oreToIngotRatio` is the correct key now. On a related note, are you planning on adding the rest of the 0.4 features, or should I pull it as is? Mostly, 0.4.x added the high oven blocks and added some book pages which can be translated.

The previous translation distorted the Russian language and was not very accurate and correct. I just fixed the inaccuracies of the previous translation. 
 And I don’t know about the keys, I did not change anything.
So you can pull it out

KnightMiner commented:
Thanks for the information on the translation.

For the keys, it's not that you changed anything, you translated from the language file in 0.3.2 (probably from Curseforge) rather from the language file in 0.4.0 (you can find that here in github), so yours was a bit outdated. I can merge this as is if you don't plan on translating the remaining keys, but it would be nice to have the language fully up to date.
